mod/groups/languages/en.php

```diff
@@ -15,8 +15,11 @@
 	'groups:delete' => 'Delete group',
 	'groups:membershiprequests' => 'Manage join requests',
 	'groups:membershiprequests:pending' => 'Manage join requests (%s)',
+	'groups:invitedmembers' => "Manage invitations",
 	'groups:invitations' => 'Group invitations',
 	'groups:invitations:pending' => 'Group invitations (%s)',
+	
+	'relationship:invited' => '%2$s was invited to join %1$s',
```

Comment:
we hardly/never use numbered variables... should we do this?

Reply:
this was the only way the have the sentence make sense. It's supported (for a long time already) so that's why i used it